### ISSUE TITLE
[doc] Awesome! How to get started? - Quick Start Readme

### ISSUE THREAD

**Issue by eabase** [hughperkins/VeriGPU#16]
This sounds really awesome! 

But how to get started from Null?

@hughperkins 
@hpasapp 

* Can you provide some more info and maybe updates on the current status on this project?
* Are you aware of any similar projects?

**Comment by eabase** [comment]
There seem to be incorrect info in:
* https://github.com/hughperkins/VeriGPU/blob/main/docs/installation.md

There is no such file at that location, and the `requirements.txt` in root is empty.

**Comment by hughperkins** [comment]
Updated the installation.md. To what extent does that work better?

**Comment by hughperkins** [comment]
> But how to get started from Null?

Yeah, running some of the tests is a great way to get started I feel. Then tinker around, try to change or add something.
